{
	"id": "oval:org.opensuse.security:def:20177478",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2017-7478",
		"affected": {
			"family": "unix",
			"platform": [
				"openSUSE Leap 42.2"
			]
		},
		"reference": [
			{
				"ref_id": "CVE-2017-7478",
				"ref_url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-7478",
				"source": "CVE"
			}
		],
		"description": "\n    OpenVPN version 2.3.12 and newer is vulnerable to unauthenticated Denial of Service of server via received large control packet. Note that this issue is fixed in 2.3.15 and 2.4.2.\n    ",
		"advisory": {
			"from": "security@suse.de",
			"severity": "Important",
			"cve": [
				{
					"text": "CVE-2017-7478",
					"href": "https://www.suse.com/security/cve/CVE-2017-7478.html"
				}
			],
			"bugzilla": [
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038713",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				},
				{
					"text": "SUSE bug 1038709",
					"href": "https://bugzilla.suse.com/CVE-2017-7478"
				}
			],
			"issued": {},
			"updated": {},
			"affected_cpe_list": {}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.opensuse.security:tst:2009157248",
								"comment": "openvpn-2.3.8-8.6.1 is installed"
							},
							{
								"test_ref": "oval:org.opensuse.security:tst:2009152221",
								"comment": "openvpn is signed with openSUSE key"
							}
						]
					},
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.opensuse.security:tst:2009157249",
								"comment": "openvpn-auth-pam-plugin-2.3.8-8.6.1 is installed"
							},
							{
								"test_ref": "oval:org.opensuse.security:tst:2009157252",
								"comment": "openvpn-auth-pam-plugin is signed with openSUSE key"
							}
						]
					},
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.opensuse.security:tst:2009157250",
								"comment": "openvpn-devel-2.3.8-8.6.1 is installed"
							},
							{
								"test_ref": "oval:org.opensuse.security:tst:2009157253",
								"comment": "openvpn-devel is signed with openSUSE key"
							}
						]
					},
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.opensuse.security:tst:2009157251",
								"comment": "openvpn-down-root-plugin-2.3.8-8.6.1 is installed"
							},
							{
								"test_ref": "oval:org.opensuse.security:tst:2009157254",
								"comment": "openvpn-down-root-plugin is signed with openSUSE key"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009135734",
				"comment": "openSUSE Leap 42.2 is installed"
			}
		]
	}
}
